content/applications/services/helpdesk/advanced/track_and_bill.rst

```diff
@@ -0,0 +1,214 @@
+click :guilabel:`New`. Add a :guilabel:`Product Name`, and set the :guilabel:`Product Type` to
+:guilabel:`Service`. Set the :guilabel:`Invoicing Policy` to :guilabel:`Prepaid/Fixed Price`. This
+means that an invoice can be generated (and payment can be received) for this product *before*
+any timesheets entries have been recorded for these services.
+
+.. image:: track_and_bill/track-bill-product-prepaid-fixed.png
+   :align: center
+   :alt: View of a service product with the invoicing policy set to 'prepaid/fixed'.
+
+Set the :guilabel:`Sales Price`, and confirm the :guilabel:`Unit of Measure` is set to
+:guilabel:`Hours`.
+
+Invoice prepaid support services
+================================
+
+When support services are billed on a fixed price, an invoice can be created before any work is
+completed on the issue. In this case, a service product with the invoicing policy set to
+:guilabel:`Prepaid/Fixed Price` (like the one created above) would be used.
+
+Create a sales order with prepaid product
+-----------------------------------------
+
+To invoice a customer for prepaid support services, first create a sales order (SO) with the
+support services product. To do this, go to :guilabel:`Odoo Dashboard --> Sales --> Orders -->
+Quotations --> New`.
+
+Fill out the :guilabel:`Quotation` with the :guilabel:`Customer` information.
+
+On the :guilabel:`Order Lines` tab, click :guilabel:`Add a Product`. Select the prepaid services
+product configured in the steps above. Update the :guilabel:`Quantity` field with the number of
+hours. After updating any other necessary information, :guilabel:`Confirm` the quotation. Then
+click :guilabel:`Create Invoice`.
+
+Create and send an invoice for prepaid services
+-----------------------------------------------
+
+This will open a :guilabel:`Create Invoices` pop-up window. If a :guilabel:`Down Payment` needs
+to be collected, change the :guilabel:`Create Invoice` type. Then click :guilabel:`Create Invoice`.
+The invoice can now be sent to the customer for payment.
+
+Create helpdesk ticket for prepaid services
+-------------------------------------------
+
+Next, navigate to the :menuselection:`Helpdesk` app and select a :guilabel:`Team`. On the
+:guilabel:`Pipeline`, click :guilabel:`New` to create a new ticket.
+
+Fill out the ticket information, and enter the :guilabel:`Customer` information. When the customer
+name is added, the :guilabel:`Sales Order Item` field will automatically populate with the most
+recent prepaid sales order item that has time remaining.
+
+Track hours on helpdesk ticket
+------------------------------
+
+Click on the :guilabel:`Timesheets` tab on the :guilabel:`Ticket`. To track the time spent on this
+ticket against the hours sold on the :abbr:`SO (sales order)`, click :guilabel:`Add a line`. Choose
+an :guilabel:`Employee`, and enter the number of :guilabel:`Hours Spent`. As new lines are added to
+tab, the :guilabel:`Remaining Hours on SO` is updated.
```

Comment:
On the *Helpdesk* ticket detail form, click into the :guilabel:`Timesheets` tab (next to the :guilabel:`Description` tab).

On the :guilabel:`Timesheets` tab, Odoo provides the ability to track the time spent on this ticket against the hours sold on the :abbr:`SO (Sales Order). To do that, click :guilabel:`Add a line` and proceed to choose an employee in the :guilabel:`Employee` column. Then, enter the number of hours spent in the :guilabel:`Hours Spent` column.

As new lines are added to the :guilabel:`Timesheets` tab, the :guilabel:`Remaining Hours on SO` field is updated automatically.